x/authority/keeper/migrator.go

```diff
@@ -20,8 +19,8 @@ func NewMigrator(keeper Keeper) Migrator {
 }
 
 // Migrate1to2 migrates the authority store from consensus version 1 to 2
-func (m Migrator) Migrate1to2(ctx sdk.Context) error {
-	return v2.MigrateStore(ctx, m.authorityKeeper)
+func (m Migrator) Migrate1to2(_ sdk.Context) error {
```

Comment:
I think these functions can be removed, but I see the older ones were still kept so I have followed the same pattern .
I will remove them if no one has any objections to it